signature/Cargo.toml

```diff
@@ -19,6 +19,7 @@ tokio = { version = "1.0", features = [ "rt", "macros", "rt-multi-thread" ] }
 tonic = "0.5"
 prost = "0.8"
 strum_macros = "0.24.2"
+sigstore_rs = { version = "0.3.3", package = "sigstore" }
```

Comment:
I notice that you've contributed to this crate. Could you get https://crates.io/crates/sigstore updated to show the repository URL? Currently, that is not showing anywhere so you need to search for it (I'm assuming it's https://github.com/sigstore/sigstore-rs :)

Reply:
Yes it is https://github.com/sigstore/sigstore-rs, and I'll update the url